app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -45,5 +73,67 @@ def patch_update
 
     private
 
+    def put_error_check
+      members = params["members"]
+
+      raise ScimRails::ExceptionHandler::InvalidPutMembers unless (members.is_a?(Array) && array_of_hashes?(members))
+
+      member_ids = members.map{ |member| member["value"] }
+
+      member_ids.each do |id|
+        @company.public_send(ScimRails.config.scim_users_scope).find(id)
+      end
+        
+      return if put_active_param.nil?
+
+      case put_active_param
+      when true, "true", 1
+        
+      when false, "false", 0
+        
+      else
+        raise ScimRails::ExceptionHandler::InvalidActiveParam
+      end
+    end
+
+    def array_of_hashes?(array)
+      array.each do |element|
+        return false unless element.is_a?(Hash)
+      end
+
+      return true
```

Comment:
No need to specifically return true/false when there are methods that return booleans already for this kind of thing.
[Check out this article](https://www.rubyguides.com/2018/10/any-all-none-one/) for more deets on iterable methods to return booleans.

There's also [reduce](https://apidock.com/ruby/Enumerable/reduce) you can check out, but stick with this one here for this particular case I think.
```suggestion
      array.all? { |hash| hash.is_a?(Hash) }
```

Reply:
I didn't know that was a thing, thanks